cmd/ci-operator/main.go

```diff
@@ -463,6 +465,20 @@ func (o *options) Complete() error {
 		}
 
 		o.clusterConfig = clusterConfig
+
+		coreGetter, err := coreclientset.NewForConfig(clusterConfig)
+		if err != nil {
+			return fmt.Errorf("could not get core client for cluster config: %v", err)
```

Comment:
If the L478 error is not blocking then this shouln't be either.

Reply:
Done.